autotest/gdrivers/tiledb_write.py

```diff
@@ -194,16 +221,27 @@ def test_tiledb_write_band_meta():
 
     src_ds = gdal.Open('../gcore/data/rgbsmall.tif')
 
-    new_ds = gdaltest.tiledb_drv.CreateCopy('tmp/tiledb_meta', src_ds)
+    for mode in [ 'BAND', 'PIXEL', 'ATTRIBUTES' ]:
```

Comment:
a more pytest'ic way would be to use pytest.parametrize (see autotest/gdrivers/fits.py for an example)